src/tests/ftest/control/dmg_storage_scan_scm.yaml

```diff
@@ -0,0 +1,6 @@
+hosts:
+  test_servers:
+    - server-A
+timeout: 102
```

Comment:
Is there a reason why the timeout is so specific?

Reply:
I set it to 1.5 times of the actual execution time. Maybe I should round up to the next 10s.